lib/pkgcloud/core/network/securityGroup.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * securityGroup.js: Base securityGroup from which all pkgcloud securityGroup inherit.
+ *
+ * (C) 2015 Rackspace
+ *      Shaunak Kashyap
+ * MIT LICENSE
+ */
+
+var util = require('util'),
+    model = require('../base/model');
```

Comment:
Sure, I'll change it here (in my code) as well as the code that existed in `lib/pkgcloud/core/network/` before this PR.